mitodl/mitxpro
# Upgrade sentry sdk

#### Pre-Flight checklist

#### What are the relevant tickets?
N/A

#### What's this PR do?
It just does an upgrade of the sentry_sdk package to hopefully address some performance issues.

#### How should this be manually tested?
Ensure you can build and run the app locally, we'll test this more thoroughly on RC.
